packages/xcache/src/index.ts

```
@@ -1,4 +1,4 @@
-export { CacheGzip } from './compress/cache-gzip';
+export { CacheCompress } from './compress/cache-compress';
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import-x/no-extraneous-dependencies)